components/ComicPanels.vue

```diff
@@ -39,6 +40,7 @@
         <swiper-slide v-for="(panel, index) in comic.getPage(0).getStrip(0).panels" :key="index">
             <WrapperCanvas
```

Comment:
I think you should be good if you just put the key already here instead of further passing the refresh-count. I don't think that it makes sense, that the `WrapperCanvas` would handle it's refresh internally.